### ISSUE TITLE
Version 10.0.17763.1697 Windows 10 LTSC  x 32

### ISSUE THREAD

**Issue by tetas26** [stascorp/rdpwrap#1286]
Can anyone with a x32 version of above help

**Comment by asmtron** [comment]
please upload the 32Bit Version of termsrv.dll 10.0.17763.1697 (x86) ....

**Comment by tetas26** [comment]
[termsrv.zip](https://github.com/stascorp/rdpwrap/files/5834325/termsrv.zip)
Thank you.

**Comment by asmtron** [comment]
add 10.0.17763.1697 (x86) to [rdpwrap.ini](https://github.com/asmtron/rdpwrap/blob/master/res/rdpwrap.ini)

[Download & Install](https://github.com/asmtron/rdpwrap/blob/master/binary-download.md)

**Comment by tetas26** [comment]
Thank you.

**Comment by HamletShaoE** [comment]
Hi, I installed the RDPwrapper, I also changed the ini file. the RDPconf.exe says all is right. But still can not have multiple people to login. I also tried to restart the computer, but RDPconf.exe. is fine but still no multiple access is allowed. Should I configure something more?
![图片](https://user-images.githubusercontent.com/4826601/106102579-9219c980-617a-11eb-88c9-b9339e3cb8e6.png)


> 
> 
> add 10.0.17763.1697 (x86) to [rdpwrap.ini](https://github.com/asmtron/rdpwrap/blob/master/res/rdpwrap.ini)
> 
> [Download & Install](https://github.com/asmtron/rdpwrap/blob/master/binary-download.md)
